### PR TITLE
treasurehunt verctorstyle

### PR DESCRIPTION
**Pull request by AFoletti**

### CODE REVIEW

**Comment by loicgasser** [comment]
If you need to we can also include icons.

**Comment by AFoletti** [comment]
yes, I need icons. But I first need to make the json layers work on the demo instance ;)

**Comment by cedricmoullet** [comment]
@AFoletti is it ready ?

**Comment by AFoletti** [comment]
no, this is still in development.
Most probably we will have to merge that only short before the golive of the new version because of the legend changes

**Comment by AFoletti** [comment]
This PR is ready to be reviewed

File: chsdi/static/vectorStyles/ch.swisstopo.treasurehunt.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "type": "unique",
+  "property": "type_coord",
+  "values": [
+    {
+      "geomType": "point",
+      "value": "link",
+      "vectorOptions": {
+        "type": "icon",
+        "src": "http://data.geo.admin.ch/ch.swisstopo.treasurehunt/2015_castor.png"
```
Comment (oterral) [inline chsdi/static/vectorStyles/ch.swisstopo.treasurehunt.json]: Use `https` if possible

File: chsdi/static/vectorStyles/ch.swisstopo.treasurehunt.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      }
+    }, {
+      "geomType": "point",
+      "value": "info",
+      "vectorOptions": {
+        "type": "icon",
+        "src": "http://data.geo.admin.ch/ch.swisstopo.treasurehunt/2015_info.png"
```
Comment (oterral) [inline chsdi/static/vectorStyles/ch.swisstopo.treasurehunt.json]: Same here

**Comment by oterral** [comment]
Are images optimized?

**Comment by AFoletti** [comment]
@oterral do you mean the legends or the icons?
For the legends, yes. The icons, I don't think so.

**Comment by oterral** [comment]
Whatever for the icon we can do it afterwards. Thx I check if the layer works then I merge.

**Comment by AFoletti** [comment]
well... @oterral forgotto merge that but it's needed for next deploy. @gjn ?

**Pull request merged**